sys/include/net/gnrc/tcp/config.h

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2015 Simon Brummer
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_tcp TCP
+ * @ingroup     net_gnrc
+ * @brief       RIOT's tcp implementation for the gnrc stack
+ *
+ * @{
+ *
+ * @file
+ * @brief       TCP configuration, includes buffersizes, timeout durations
+ *
+ * @author      Simon Brummer <simon.brummer@haw-hamburg.de>
+ * @}
+ */
+
+#ifndef GNRC_TCP_CONFIG_H_
+#define GNRC_TCP_CONFIG_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Status Flags for TCP
+ */
+#define GNRC_TCP_STATUS_PASSIVE (1 << 0)
+#define GNRC_TCP_STATUS_ACCEPTED (1 << 1)
+#define GNRC_TCP_STATUS_ALLOW_ANY_ADDR (1 << 2)
+
+/**
+ * @brief Timeout Duration for user calls. Default 2 minutes
+ */
+#ifndef GNRC_TCP_USER_TIMEOUT_DURATION
+#define GNRC_TCP_USER_TIMEOUT_DURATION (2 * 60 * 1000 * 1000)
+#endif
```

Comment:
Please use `SEC_TO_USEC`; or alternatively it might be even better to set all timeouts in seconds only and convert when setting the (x)timer